modules/operator-unmanaged-storage.adoc

```diff
@@ -57,23 +57,58 @@ DISTRIBUTED_STORAGE_PREFERENCE:
 +
 As of {productname} 3.7, you must use the Primary endpoint of your MAG Blob service. Using the Secondary endpoint of your MAG Blob service will result in the following error: `AuthenticationErrorDetail:Cannot find the claimed account when trying to GetProperties for the account whusc8-secondary`.
 
-== Ceph / RadosGW Storage / Hitachi HCP storage 
+== RadosGW Storage
+[source,yaml]
+----
+DISTRIBUTED_STORAGE_CONFIG:
+  radosGWStorage: #storage config name
+    - RadosGWStorage #actual driver
+    - access_key: access_key_here #parameters
+      secret_key: secret_key_here
+      bucket_name: bucket_name_here
+      hostname: hostname_here
+      is_secure: 'true'
+      port: '443'
+      storage_path: /datastorage/registry
+DISTRIBUTED_STORAGE_DEFAULT_LOCATIONS: []
+DISTRIBUTED_STORAGE_PREFERENCE: #must contain name of the storage config
+    - radosGWStorage
+----
 
+== Ceph Storage
 [source,yaml]
 ----
 DISTRIBUTED_STORAGE_CONFIG:
-  radosGWStorage:
-    - RadosGWStorage
-    - access_key: access_key_here
+  cephStorage: #storage config name
+    - CephStorage #actual driver
+    - access_key: access_key_here #parameters
       secret_key: secret_key_here
       bucket_name: bucket_name_here
       hostname: hostname_here
       is_secure: 'true'
       port: '443'
       storage_path: /datastorage/registry
 DISTRIBUTED_STORAGE_DEFAULT_LOCATIONS: []
-DISTRIBUTED_STORAGE_PREFERENCE:
-    - default
+DISTRIBUTED_STORAGE_PREFERENCE: #must contain name of the storage config
+    - cephStorage
+----
+
+== Hitachi HCP storage
+[source,yaml]
+----
+DISTRIBUTED_STORAGE_CONFIG:
+  hcpStorage: #storage config name
+    - HCPStorage #actual driver
+    - access_key: access_key_here #parameters
+      secret_key: secret_key_here
+      bucket_name: bucket_name_here
+      hostname: hostname_here
+      is_secure: 'true'
+      port: '443'
+      storage_path: /datastorage/registry
+DISTRIBUTED_STORAGE_DEFAULT_LOCATIONS: []
+DISTRIBUTED_STORAGE_PREFERENCE: #must contain name of the storage config
+    - hcpStorage
```

Comment:
Same comment as before. We don't have a specific Hitachi driver. All drivers that are S3 compatible can be found here:

https://github.com/quay/quay/blob/ff66a93eb7c1b466b2dffd5e62187d7824e6ccad/storage/__init__.py#L21

These are storage classes that we have in Quay and we support.